test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -437,4 +437,32 @@ spec =
             Meta.type_of (_.is_nothing) . should_equal Function
             Meta.type_of (.is_nothing) . should_equal Function
 
+    Test.group "Meta.Type.find"
+        Test.specify "find boolean and use it" <|
+            fqn = "Standard.Base.Data.Boolean.Boolean"
+            typ = Meta.Type.find fqn
+            typ . is_a Meta.Type . should_be_true
+            typ.name . should_equal "Boolean"
+            Meta.Type.find typ.qualified_name . should_equal typ
+
+        Test.specify "find boolean via associated type and use it" <|
+            fqn = "Standard.Base.Data.Boolean"
+            typ = Meta.Type.find fqn
+            typ . is_a Meta.Type . should_be_true
+            typ.name . should_equal "Boolean"
+            typ.qualified_name . should_equal fqn
+            Meta.Type.find typ.qualified_name . should_equal typ
+
+        Test.specify "unknown type" <|
+            Meta.Type.find "Unknown.Type" . should_fail_with Not_Found
+            Meta.Type.find "UnknownType" . should_fail_with Not_Found
+            Meta.Type.find "" . should_fail_with Not_Found
+
+        Test.specify "My ascribed type" <|
+            meta_type = Meta.meta My_Ascribed_Type
+            fqn =  meta_type . qualified_name
+            "Meta_Spec.My_Ascribed_Type" . should_equal fqn
```

Comment:
We usually try to keep the format `got.should_equal expected`.

```suggestion
            fqn . should_equal "Meta_Spec.My_Ascribed_Type"
```

Reply:
Changed in d536ea7 to search for a substring only.